src/Engine/ProtoCore/Reflection/Mirror.cs

```diff
@@ -416,8 +416,11 @@ public static IEnumerable<ClassMirror> GetClasses(Core core)
 
             public static IEnumerable<ClassMirror> GetAllTypes(Core core)
             {
+                // TODO: Get rid of keyword "PointerReserved" and PrimitiveType.kTypePointer
+                // if not used in the language: http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6752
                 return core.ClassTable.ClassNodes.
-                    Where(x => !CoreUtils.StartsWithSingleUnderscore(x.name)).
+                    Where(x => !CoreUtils.StartsWithSingleUnderscore(x.name)
+                    && x.name != DSDefinitions.Keyword.PointerReserved).
```

Comment:
We can try to potantially just remove PointerReserved in a future refactor work.
In fact ill add it to the enhancements task: http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6193